guides/common/modules/proc_viewing-multiple-content-view-environment-activation-keys.adoc

```diff
@@ -0,0 +1,38 @@
+[id="Viewing_multiple_content_view_environment_activation_keys_{context}"]
+= Viewing multiple content view environment activation keys
+
+Use this procedure to view activation keys with multiple content view environments or multi-environment activation keys in the {ProjectWebUI} and Hammer CLI.
+
+.Prerequisites
+* Ensure you have activation keys created.
+For more information, see xref:Creating_an_Activation_Key_{context}[Creating an activation key].
+
+.Procedure
+. In the {ProjectwebUI}, navigate to *Content* > *Lifecycle* > *Activation Keys*.      
+. Select an activation key.
+. Under the *Details* tab, the *Content view environments* are displayed.
+
+[NOTE]
+====
+You cannot reassign a multi-environment activation key's content view environments using the web UI.
+To change a multi-environment activation key's content view environments, update the activation key with Hammer.
```

Comment:
While technically accurate, I think it's too much of a mouthful and we could remove some words:

```suggestion
To change content view environments, update the activation key with Hammer.
```